grails-app/services/org/bbop/apollo/TranscriptService.groovy

```diff
@@ -470,4 +474,145 @@ class TranscriptService {
     Transcript getTranscript(CDS cds) {
         return (Transcript) featureRelationshipService.getParentForFeature(cds, ontologyIds as String[])
     }
+
+    /**
+     * @param gsolFeature
+     * @param includeSequence
+     * @return
+     */
+    @Timed
+    JSONObject convertTranscriptToJSON(Transcript gsolFeature, List<Feature> childFeatures = new ArrayList<>(),List<FeatureLocation> childFeatureLocations = new ArrayList<>()) {
```

Comment:
@cmdcolin   As to your other comment, passing in the child feature locations removed a SIGNIFICANT amount of re-querying.   True, this won't work if more than one FeatureLocation, but that is kind of a global problem to solve.